tslint.json

```diff
@@ -123,6 +107,29 @@
       "check-separator",
       "check-type"
     ],
-    "import-destructuring-spacing": true
+    "directive-selector": [
+      true,
+      "attribute",
+      "ccl",
+      "camelCase"
+    ],
+    "component-selector": [
+      true,
+      "element",
+      "ccl",
+      "kebab-case"
+    ],
+    "use-input-property-decorator": true,
+    "use-output-property-decorator": true,
+    "use-host-property-decorator": true,
+    "no-input-rename": true,
+    "no-output-rename": true,
+    "use-life-cycle-interface": true,
+    "use-pipe-transform-interface": true,
+    "component-class-suffix": true,
+    "directive-class-suffix": true,
+    "no-access-missing-member": true,
+    "templates-use-public": true,
+    "invoke-injectable": true
```

Comment:
in all honesty i dont know what all these settings are but if they're defaults from ng-cli, that seems good to me since that's the standard convention